examples/htlc-pyteal-ts/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "./build/scripts"
+    "outDir": "./scripts"
```

Comment:
This would be a workaround to get the example working. But i am not sure if we should mix `/build` files in root. 

As i think it will complicate the current structure (have multiple files in root). Also, if the user decides to add one more file (say `scripts/1-deploy.ts`) then, for each file the `build` will generate 3 extra files (`.js, .d.ts, .js.map`). And if the above suggestion doesn't work, user will also need to manually edit `.gitignore` as well. So i don't think this would be a good user & developer experience.

Did you check why the deployer is not loading the file from `/build/scripts`? Particularly this line: [https://github.com/scale-it/algorand-builder/blob/master/packages/algob/src/builtin-tasks/deploy.ts#L35](https://github.com/scale-it/algorand-builder/blob/master/packages/algob/src/builtin-tasks/deploy.ts#L35). Interestingly, when i was trying to run the example, i tried to add a `console.log()` after `executeDeployTask` was called (to check what's happing). But it didn't show the log when i ran `yarn run algob deploy` in `/examples`. I don't know why, but i am unable to debug because of this. Can you check on your system as well?

Reply:
working example would be a good user experience, what do you suggest to resolve this issue  ?

Reply:
nvm